### ISSUE TITLE
Support multiple lineups in custom channel configuration

### ISSUE THREAD

**Issue by h3298** [edit4ever/script.module.zap2epg#7]
In "configure channel list", support picking channels from multiple zap2it lineups (example: one channel comes from cable lineup, another comes from satellite lineup)

**Comment by edit4ever** [comment]
This will take a bit of reworking as the system will have to download complete data for both lineups.  I might be able to work on this in the next month or so - but no promises.  As an FYI, Schedules Direct works really well for this - and in my opinion for complicated (or just reliable) guide info - is well worth the $25/yr.

I'll keep this open for now to remind me to work on it.

**Comment by h3298** [comment]
Yeah, I realized that kind of structural change is probably not a quick fix.  I just put it out there as an idea for long term expansion of the feature set.  There are a surprising number of free IPTV channels that started out as cable or satellite channels, so thats where the EPG data is.  I've been trying to use that data to build a legal IPTV setup with a fully functioning guide using only what is freely available on the web, to demonstrate legitimate uses for Kodi that cannot be disputed.  It's fine to reference Schedules Direct for the sake of those who may be interested, but as we have seen so many services come and go, I think it's best not to put all your eggs in one basket.  I actually spend more time tracking down bugs in Kodi and trying to improve the UI than I do on watching TV.  =)

**Comment by wbeard52** [comment]
I was able to accomplish downloading different zap2it schedules by copying the KODI addon folder and changing its name on the folder and in the XML files.  I also needed to make a copy of the wrapper file that is called by TVH and change it to point to the new Kodi addon location.  With a restart, TVH saw the newly created wrapper and i then continued to set up the second zap2it addon for the other schedule.  It works great

**Comment by edit4ever** [comment]
You basically created a duplicate app...so it makes sense that it works.  Not the most efficient - but very creative solution.  :-)  I haven't worked on this in a long time as it has been stable and there have been other projects that have taken my focus.  I may not get to the multiple lineups, since it is a small universe of people who need it - but thanks for the idea for a workaround for others!

**Comment by esenterre** [comment]
I'm in a similar situation. I mainly use my OTA lineup, but I configured one IPTV channel in TVH that is on another lineup. Would have been nice to have that feature in zap2epg. But I understand that's maybe too much work to do.  

@wbeard52 has a solution, but looks a bit complicated to me.

Thanks for all the good work!

**Comment by hagertyoh** [comment]
I basically did what @wbeard52 indicated above and placed in a zip file located below:  https://github.com/hagertyoh/script.module.zap2epg/archive/refs/heads/zap2epg2.zip

if you load the zip file it will show addon zap2epg2. it is identical to the originalzap2epg with only changed a few names to allow a second instance.
